addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14143,19 +14153,19 @@ msgstr ""
 #. Description of setting "Appearance -> File lists -> Show file extensions" with label #497
 #: system/settings/settings.xml
 msgctxt "#36123"
-msgid "Show file extensions on media files. For example, 'You Enjoy Myself.mp3' would be simply be shown as 'You Enjoy Myself'."
+msgid "Show file extensions on media files (e.g. \"You Enjoy Myself.mp3\" will simply be shown as \"You Enjoy Myself\".)"
 msgstr ""
 
 #. Description of setting "Appearance -> File lists -> Ignore articles when sorting with label #13399
 #: system/settings/settings.xml
 msgctxt "#36124"
-msgid "Ignore certain tokens (e.g. \"the\") during sort operations. For example, 'The Simpsons' would be sorted as 'Simpsons'."
+msgid "Ignore certain tokens (e.g. \"The\") during sort operations, e.g. \"The Simpsons\" will be sorted as \"Simpsons\"."
```

Comment:
Well atm they are all e.g. no braces there are no for examples anywhere anymore.

I would need to go dig back into the unchanged file to find them and put them back.